### ISSUE TITLE
Suggestion: CLI command for `new post` with auto date format?

### ISSUE THREAD

**Issue by peterkos** [Keats/gutenberg#1855]
Hello!

One thing that I'd find useful is to have a command for making new blog posts that are auto-timestamped. Something like this, where the date/timestamp are auto generated from the current point:
```shell
> zola new 
>| Title: CoolTitle
~Created page in content/2022-05-11-CoolTitle.md~
```

And then it can also fill in the header:
```markdown
// content/2022-05-11-CoolTitle.md
+++
title = "CoolTitle"
date = 2022-05-11
+++
```

**Comment by JeremyRudman** [comment]
I think that is a pretty poggers idea 👍

**Comment by Keats** [comment]
This has been mentioned a few times but this is not going to happen

**Comment by bemyak** [comment]
I made proposal to enable this in a form of Zola plugin: https://zola.discourse.group/t/proposal-cargo-style-extensions/1089

**Comment by peterkos** [comment]
Is there any particular reason why this can't happen, if its been mentioned a few times?

I've gone ahead and written it for my own use on [a fork](https://github.com/peterkos/zola/tree/zola-1855). I'm happy to add some tests if its decided to have as part of the main project.

**Comment by Keats** [comment]
> Is there any particular reason why this can't happen, if its been mentioned a few times?

It's strictly inferior to copy/pasting an existing md file as it's not going to be flexible enough to cover even slightly complex usecases. Then you end up re-implementing Hugo archetypes to make up for the lack of flexibility and you end up with something still more complex than copy/paste.

**Comment by alebaffa** [comment]
Finding this 3 years later, because it was the **very** first thing I saw missing in this project when I tried it few days ago 😆
 
TL;DR: 
users must remember by heart all the possible frontmatter metadata available for posts and pages, and add them manually each time (or make a .sh script to generate them).

**Comment by bemyak** [comment]
I still maintain the [zola-new](https://gitlab.com/bemyak/zola-new) project, it should be up-to-date with Zola. Feel free to use it :)

**Comment by alebaffa** [comment]
> I still maintain the [zola-new](https://gitlab.com/bemyak/zola-new) project, it should be up-to-date with Zola. Feel free to use it :)

Thanks! I also found this one: https://github.com/Geobert/emile.
